deploy/gcp-compute-persistent-disk-csi-driver-custom-role.yaml

```diff
@@ -4,3 +4,4 @@ stage: ALPHA
 includedPermissions:
 - compute.instances.get
 - compute.instances.attachDisk
+- compute.instances.detachDisk
```

Comment:
does the e2e not test this?

Reply:
e2e currently doesn't test service account  because thats kind of "out-of-band." I guess I could have it invoke the setup scripts and pull the environment variables for Service Account just like everywhere else.... Now that I write that it seems like a good idea.

https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/75